<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomersWindow</class>
 <widget class="QDialog" name="CustomersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="addCustomerButton">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/icons/Resources/user-add.png</normaloff>:/new/icons/Resources/user-add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidgetCustomers">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="Active">
      <attribute name="title">
       <string>Active</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLineEdit" name="searchLE">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Search:</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTableView" name="tableViewCustomersActive">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxCustomersActive">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string comment="Select field to sort:" extracomment="Select field to sort:">Select field to sort:</string>
         </property>
         <item>
          <property name="text">
           <string>customer_id</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>surname</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>firstName</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>middleName</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>age</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>phoneNumber</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>email</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yearsOfExperience</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>itin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>address</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Closed">
      <attribute name="title">
       <string>Closed</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QTableView" name="tableViewCustomersClosed">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="searchLE2">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxCustomersClosed">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string comment="Select field to sort:" extracomment="Select field to sort:">Select field to sort:</string>
         </property>
         <item>
          <property name="text">
           <string>customer_id</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>surname</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>firstname</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>middleName</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>age</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>phoneNumber</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>email</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yearsOfExperience</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>itin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>address</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
